src/pages/Rooms.tsx

```diff
@@ -86,6 +86,10 @@ function join(
   ]
 }
 
+const RoomNameStyled = styled.b`
```

Comment:
```suggestion
const RoomName = styled.span`
```
See above for the naming thing. I think using a `b` tag was a mistake in the first place as it doesn't carry any semantic meaning.